### PR TITLE
Changelog for 1.3

### PR DESCRIPTION
**Pull request by wild-endeavor**
Just the changelog, no code changes.

### CODE REVIEW

File: CHANGELOG/CHANGELOG-v1.3.0.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+   default_project="flytesnacks",
+   default_domain="development",
+)
+r.list_signals("atc526g94gmlg4w65dth")
+r.set_signal("signal-name", "execidabc123", True)
+```
+
+## Looking Ahead
+In the coming release, we are focusing on...
+1. [Out of core plugin](https://unionai.atlassian.net/wiki/spaces/FLYTE/pages/47153191/Out+of+core+plugins+Prove+Value): Make backend plugin scalable and easy to author. No need of code generation, using tools that MLEs and Data Scientists are not accustomed to using.
```
Comment (hamersaw) [inline CHANGELOG/CHANGELOG-v1.3.0.md:99]: @pingsutw is this doc restricted to the union team in confluence? if so we probably shouldn't link to it in the public release notes.

**Comment by hamersaw** [comment]
@wild-endeavor how recent versions of everything do we use to cut the release? Nicks cache overwrite stuff should be in end to end now.

File: CHANGELOG/CHANGELOG-v1.3.0.md (hunk 1 above)
Comment (pingsutw) [reply CHANGELOG/CHANGELOG-v1.3.0.md]: updated the link, sorry, I thought that is public

**Comment by wild-endeavor** [comment]
it's whenever we run the release job @hamersaw .  so whatever the active version is of all the components as of maybe later this afternoon.

**Review by eapolinario — APPROVED** [review]

**Pull request merged**